requirements.txt

```diff
@@ -8,7 +8,8 @@ s3fs==0.4.2
 gcsfs==0.6.2
 outdated==0.2.0
 lz4>=3,<4
-zarr
+zarr=2.4
+lmdb==1.0.0
```

Comment:
This is very cool, because we are not limited with RAM but with HDD allowing more caching. Still I think we will need a smaller RAM cache to make it even faster. But for now I think this will do.

Reply:
lmdb should have the option to store everything in memory, I can investigate further.